app/graphql/users/resolver.js

```diff
@@ -0,0 +1,19 @@
+const { user: User } = require('../../models'),
+  errors = require('../../errors'),
+  logger = require('../../logger'),
+  helpers = require('../../helpers');
+
+exports.createUser = async user => {
+  try {
+    const hash = await helpers.encryptPassword(user.password);
+    user.password = hash;
```

Comment:
assign it directly: `user.password = await helpers.encryptPassword(user.password);`